### PR TITLE
Feature/package zuericssstyle

### PR DESCRIPTION
**Pull request by alfell**
Updated LIMA to zuericssstyle.

### CODE REVIEW

File: app.R [hunk 1]
```diff
@@ -39,14 +40,14 @@ if (is.null(data)) {
   ui <- fluidPage(
 
     # CSS
-    includeCSS("sszTheme.css"),
+    includeCSS("sszThemeShiny.css"),
 
     # App Selection
     tags$div(
-      class = "radioDiv",
+      class = "queryDiv",
```
Comment (shaenzi) [inline app.R:47]: compare with zuericssstyle: the function for the radio buttons already wraps the radio buttons in a [radioDiv](https://github.com/StatistikStadtZuerich/zuericssstyle/blob/main/R/ssz_ui_utils.R#L263), so no need to have this here. also I do not see a `queryDiv` anyway

**Review by shaenzi — COMMENTED** [review]
queryDiv gefällt mir nicht ( siehe separater Kommentar)

File: app.R (hunk 1 above)
Comment (alfell) [reply app.R:52]: We need `queryDiv` only LIMA since the first two Radio Buttons for the initial query should look different than the rest.

**Review by shaenzi — CHANGES_REQUESTED** [review]
OGD icon missing in second app

**Review by shaenzi — APPROVED** [review]

**Pull request merged**